spec/requests/user/user_suggestions_spec.rb

```diff
@@ -1,69 +1,100 @@
 require "rails_helper"
 
 RSpec.describe "Users", type: :request do
-  describe "GET /users" do
-    it "returns no users if state is not present" do
-      get users_path
-
-      expect(response).to have_http_status(:ok)
+  RSpec.shared_examples "a default suggested_users request" do |path|
+    let(:suggested_users_list) { %w[eeyore] }
+    let!(:suggested_user) { create(:user, name: "Eeyore", username: "eeyore", summary: "I am always sad :(") }
 
-      expect(response.parsed_body).to be_empty
+    before do
+      allow(SiteConfig).to receive(:suggested_users).and_return(suggested_users_list)
     end
 
-    it "returns users from the original core team if they are present" do
-      user = create(:user, username: "ben", summary: "Something something")
-
-      get users_path
+    it "returns the default suggested_users from SiteConfig if they are present" do
+      get path
 
       expect(response).to have_http_status(:ok)
 
       response_user = response.parsed_body.first
-      expect(response_user["id"]).to eq(user.id)
-      expect(response_user["name"]).to eq(user.name)
-      expect(response_user["username"]).to eq(user.username)
-      expect(response_user["summary"]).to eq(user.summary)
-      expect(response_user["profile_image_url"]).to eq(ProfileImage.new(user).get(width: 90))
+      expect(response_user["id"]).to eq(suggested_user.id)
+      expect(response_user["name"]).to eq(suggested_user.name)
+      expect(response_user["username"]).to eq(suggested_user.username)
+      expect(response_user["summary"]).to eq(suggested_user.summary)
+      expect(response_user["profile_image_url"]).to eq(ProfileImage.new(suggested_user).get(width: 90))
       expect(response_user["following"]).to be(false)
     end
+  end
 
-    it "returns follow suggestions for an authenticated user" do
-      user = create(:user)
-      tag = create(:tag)
-      user.follow(tag)
+  describe "GET /users" do
+    let(:user) { create(:user) }
 
-      other_user = create(:user)
-      create(:article, user: other_user, tags: [tag.name])
+    context "when the user is not signed in" do
+      it_behaves_like "a default suggested_users request", "/users"
+    end
 
-      sign_in user
+    context "when no state params are present" do
+      before { sign_in user }
```

Comment:
Cleaned up these specs quite a bit and added some shared examples (🎉!!!). Notably, I added two specific contexts for when we `sign_in user` versus when we don't (since we have a path for that scenario [here](https://github.com/thepracticaldev/dev.to/pull/7637/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R16)).

Reply:
Update: I had to remove the shared examples; for some (currently inexplicable to me) reason, shared examples caused failures on CI that I couldn't repro locally 🤷